src/DataTable/TableActions.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import DataTableContext from './DataTableContext';
+import Actions from './Actions';
+
+const TableActions = ({
```

Comment:
I know we discussed this name before, but I wonder if `Actions` above is better named `TableActions` and this component `StaticActions` or `DefaultActions`.